client/my-sites/subscribers/subscriber-list/styles.scss

```diff
@@ -0,0 +1,125 @@
+@import "@automattic/color-studio/dist/color-variables";
+@import "@wordpress/base-styles/breakpoints";
+
+.subscriber-list {
+	margin: 0;
+
+	.row {
+		border-block-end: 1px solid $studio-gray-5;
+		display: flex;
+		align-items: center;
+		flex-direction: row;
+		padding-top: 20px;
+		padding-bottom: 20px;
+
+		* {
+			flex: 1;
+		}
+
+		.hidden {
+			display: none;
+		}
+
+		&.header {
+			padding-bottom: $font-code;
+			padding-top: 0;
+
+			@media (max-width: $break-small) {
+				display: none;
+			}
+		}
+
+		.subscriber-list__checkbox-column,
+		.subscriber-list__profile-column,
+		.subscriber-list__subscription-type-column,
+		.subscriber-list__rate-column,
+		.subscriber-list__since-column,
+		.subscriber-list__menu-column {
+			font-weight: 400;
+			font-size: $font-body-small;
+			font-size: rem(14px);
```

Comment:
The `font-size` is set two times here. I believe you wanted to keep `font-size: $font-body-small;` only.